src/Bridge/Symfony/Resources/config/twig.xml

```diff
@@ -11,6 +11,7 @@
         <service id="sonata.twig.status_extension" class="Sonata\Twig\Extension\StatusExtension">
             <tag name="twig.extension"/>
         </service>
+        <!-- NEXT_MAJOR: Remove this definition. -->
         <service id="sonata.twig.deprecated_template_extension" class="Sonata\Twig\Extension\DeprecatedTemplateExtension">
             <tag name="twig.extension"/>
```

Comment:
```suggestion
            <deprecated>The "%service_id%" service is deprecated since sonata-project/twig-extensions 1.x.</deprecated>
            <tag name="twig.extension"/>
```